src/components/CodeSearchResults/CodeSearchResultsTable.jsx

```diff
@@ -88,28 +115,34 @@ const CodeSearchResultsTable = ({
         <Icon className="fa fa-check text-primary" screenReaderText="has been redeemed" />
       ) : defaultEmptyValue,
       courseTitle: courseTitle || defaultEmptyValue,
+      assignedTo: assignedTo || defaultEmptyValue,
       redemptionDate: redemptionDate || defaultEmptyValue,
       actions: !isRedeemed ? (
         <React.Fragment>
-          <RemindButton
-            couponId={couponId}
-            couponTitle={couponName}
-            data={{
-              email: searchQuery,
-              code,
-            }}
-            onSuccess={onRemindSuccess}
-          />
-          {' | '}
-          <RevokeButton
-            couponId={couponId}
-            couponTitle={couponName}
-            data={{
-              assigned_to: searchQuery,
-              code,
-            }}
-            onSuccess={onRevokeSuccess}
-          />
+          {isAssigned ? (
```

Comment:
This is now a nested ternary, which can be a bit difficult to interpret quickly. Is there a way to avoid the nested ternary? Perhaps:
```
actions: !isRedeemed && isAssigned ? (
  <React.Fragment>
    <RemindButton />
    <RevokeButton />
  </React.Fragment>
) : defaultEmptyValue
```

Also, if a code is neither redeemed/assigned, should the search results table display an "Assign" button similar to how the coupon details table behaves? This would allow assigning of codes directly from the search results. For example, an enterprise admin searches for code XYZ, sees it's not assigned to anyone, and then assigns it directly from the search results instead of finding within one of the coupons below the search results.

Reply:
Yes, the nested ternary would make the code less readable. I have fixed this issue in the latest commit. Thanks for pointing that out.
I would also agree with you on the second point regarding the Assign button. I have discussed this scenario with @georgebabey and as per our discussion, this is a feature enhancement which will be covered later in the future.